README.md

```diff
@@ -54,9 +54,10 @@ means = np.array([[0,0], [-5,5], [5,5]])
 X = np.vstack([np.random.randn(n_per_cluster, 2) + mu for mu in means])
 
 # Fit the data with BanditPAM:
-kmed = KMedoids(n_medoids = 3, algorithm = "BanditPAM")
+k = 3
```

Comment:
ordinarily it's good practice to define variables and avoid magic numbers, but in this case for the README examples, since `k` is only used once, let's not use a separate variable and just pass `3` in the call below